### ISSUE TITLE
lerna version --no-tag is either needs readme clarification or is buggy

### ISSUE THREAD

**Issue by revelt** [lerna/lerna#1993]
## Expected Behavior

`lerna version` documentation says:

> By default, lerna version will push the committed and tagged changes to the configured git remote. Pass --no-push to disable this behavior.
> https://github.com/lerna/lerna/tree/master/commands/version#--no-push

I noticed that when this behaviour is disabled, tags are not created at all.

Now, from logical standpoint, either documentation needs to say something like:

> By default, lerna version will **create and** push the committed and tagged changes to the configured git remote. Pass --no-push to **neither create not push** any tags at all.

Or, **this is a bug** (which I hope it is because I want those git tags, I just don't want them pushed at that time).

## Some context

I want to push my monorepo to a remote (on GitLab) and its CI should run `lerna version`, update all files and publish to npm. Then, CI should commit whatever was changed, to the same remote, as if somebody did the release for me. What's left is only `git pull` on my computer.

I achieved all the above except git tags are missing.

Now, obvious direction is to remove `--no-push`, but since action is happening on CI already, there's no need to push **at this step**. I will push everything in the end anyway, and importantly, with `[skip ci]` commit message which will skip that push causing an infinite loop (every push starts a build).

Maybe I'm doing something not right, but documentation is scarce and I don't see many people publishing npm libraries using lerna monorepos from CI. If you know any, please leave a comment, I'd gladly reference their setup.

## Current Behavior
Currently, `lerna version --no-push` will not create any tags at all.

## Possible Solution
If possible, either clarify the documentation to state that git tags will not be created and will not be pushed, OR fix the lerna to create the tags but not push them.

## Steps to Reproduce (for bugs)

I called the following npm script from GitLab CI:

```
lerna version --conventional-commits --no-commit-hooks --no-push --yes
```

For reference, here's my repo illustrating the situation, from this morning's release: https://gitlab.com/codsen/codsen/commit/c00a72a42dcbedefccb122bdd4396f03117ef245
Git tags are missing.

<details><summary>lerna.json</summary><p>

```json
{
  "command": {
    "publish": {
      "conventionalCommits": true,
      "message": "chore: Release"
    }
  },
  "lifecycle": {
    "version": "npm run build && git add ."
  },
  "packages": [
    "packages/*"
  ],
  "version": "independent"
}
```
</p></details>

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.13.1 |
| `npm --version`  | 6.4.1 |
| `yarn --version` | n/a, but 1.12.3 |
| `node --version` | 10.15.3 |

| OS | Version |
| --- | --- |
| macOS Sierra | 10.14.3 |

**Comment by tgrux** [comment]
@revelt - I'm doing something similar and used the lerna.json to add the [skip-ci] to the default lerna comment message
`"command":{"version": {"message": "my new version %s [ci skip]"}}`

That might help you continue to use the CI to do this and not have an loop of pipelines.

**Comment by revelt** [comment]
@tgrux thank you. Long time has passed since I posted originally; I ended up using Lerna to only bump all `package.json`'s and publish via `lerna run` + npm publish script on a premise that same version packages will stop from being published... I'm happy with my (inefficient) [setup](https://gitlab.com/codsen/codsen/-/blob/master/.gitlab-ci.yml) — I'm happy to close this issue unless anybody sees value in it. Just let me know and I'll close, or maintainers can close right away.
